esmvalcore/preprocessor/_other.py

```diff
@@ -58,6 +59,88 @@ def clip(cube, minimum=None, maximum=None):
     return cube
 
 
+@preserve_float_dtype
+def cumsum(
```

Comment:
`cumulative_sum` would be more descriptive, would you be open to changing the name?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2642/commits/548a7101b32b5b8ba62e57fdc8f2902234c1e301.